wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,276 @@
 
-**Note:** While Half Time and Double Time do not change the actual OD value, the speed's effect on hit windows will make circles seem to have a lower or higher apparent OD, respectively. In the chart below, apparent OD values are provided to allow comparison between timings with and without these mods. Note that these apparent OD values only apply to 300s on circles. Windows for 100s, 50s, and sliders (which use the 50 hit window) scale more harshly with Double Time than this apparent OD suggests, and more leniently with Half Time. Again, outside of NoMod/HR/EZ (on the left side), the OD values below are only for comparison: the actual OD value is always a number from 0 to 10.
+#### Sample Set Selection
 
-![Detailed OD chart](img/ODTable.png "Detailed OD chart")
+Choose to use `Normal`, `Soft`, or `Drum` built-in sample set.
 
-**Note:** The osu! timing system does not allow a [hit circle](/wiki/Hit_Objects) to be hit until the previous one has been hit or its time frame has been exceeded (resulting in a miss). With a low OD, the time frame of one circle may overlap with the next. Thus, one could hit the second note with perfect timing (after failing to hit the first note) and end up completely missing both because the time frame of the first note has not been exceeded yet.
+It is possible to [enable custom overrides][Custom Sample Overrides] (put hit sounds files inside the beatmap's folder first) for custom hit sounds instead.
 
-### Approximate Difficulty Rating
+#### Sample Set Volume
 
-This is a summary of all of the settings chosen on this page. More stars mean harder maps and more [score](/wiki/Score). This is not the final star ranking of the song; it is just an approximation based on the settings you chose.
+Set the volume of the sample set.
 
-Audio
------
+#### Test Sample Set
 
-![Audio section](img/SS_Audio.jpg "Audio section")
+Click on the buttons to test out how the sample set sounds. Four buttons are provided, namely: `Normal`, `Whistle`, `Finish`, and `Clap`.
 
-### Default Sample Settings
+These sounds are additive; `Whistle` is a combination of `Normal` and `Whistle`.
 
-Here you can configure the hit sounds to your liking.
+### Misc. Toggles
 
-If you have [timing](/wiki/Beatmap_Editor/Timing) sections (F6) that change either the hit-sound set or volume, you will not be able to adjust them here. You can click reset settings to remove them. Most mappers use timing sections to set the settings you can find here.
+Miscellaneous toggles.
 
-#### Sample Set Selection
+`Samples match playback rate (for fully-hitsounded maps)` checkbox will adjust the hit sounds' pitch accordingly to the speed modifiers. Unchecking it will let the hit sounds play at its own preset pitch. This option is subjective to the creator so please use discretion before checking it.
```

Comment:
This should be changed a bit based off reading this https://osu.ppy.sh/forum/p/6429837